### ISSUE TITLE
Create a list of IOOS docs pages in the README

### ISSUE THREAD

**Issue by ocefpaf** [ioos/.github#8]
These are the pages I found by searching `sync_theme.yml` in the IOOS org:

1. https://github.com/ioos/conventions-for-observing-asset-identifiers
1. https://github.com/ioos/glider-dac
1. https://github.com/ioos/ioos-documentation-jekyll-skeleton (skeleton)
1. https://github.com/ioos/ioos-metadata
1. https://github.com/ioos/ioos.github.io
1. https://github.com/ioos/ioosngdac (archived)
1. https://github.com/ioos/mbon-docs
1. https://github.com/ioos/ncei-archiving-cookbook
1. https://github.com/ioos/erddap-gold-standard
1. https://github.com/ioos/passive-acoustics
1. https://github.com/ioos/BioData-Training-Workshop
1. https://github.com/ioos/biological-data-services
1. https://github.com/ioos/sos-guidelines
1. https://github.com/ioos/ioos-csv-tsv
1. https://github.com/ioos/ioos-atn-data

With the exception of the ioosngdac, that is archived, and ioos-documentation-jekyll-skeleton, that is an example, all others are good candidates for this list (6 total).

**Comment by mwengren** [comment]
Here are a few more:

1. https://github.com/ioos/erddap-gold-standard
2. https://github.com/ioos/passive-acoustics
3. https://github.com/ioos/BioData-Training-Workshop
4. https://github.com/ioos/biological-data-services  
5. https://github.com/ioos/sos-guidelines
6. https://github.com/ioos/ioos-csv-tsv
7. https://github.com/ioos/ioos-atn-data

That's all I can come up with.  I think several of those in my list we need to re-enable the GHA to pull in recent commits.  I'll do that now.  

@MathewBiddle I forget what our plan was with number 4 above.  Should we just unpublish that site?  I don't think it's linked from anywhere.

**Comment by mwengren** [comment]
**Note**: to re-enable a timed-out GHA such as 'sync-theme.yml', go to the repo in question, navigate to Actions > sync theme submodules, hit 'Enable workflow'.

**Comment by ocefpaf** [comment]
@mwengren then we need to re-run the workflow using the dispatch:

![Screenshot from 2023-09-19 11-34-07](https://github.com/ioos/.github/assets/950575/b5f45968-124a-4bf7-b182-5a339d660a4b)


PS0: Do you mind if I copy your list from https://github.com/ioos/.github/issues/8#issuecomment-1723681568 into the my list above? Feel free to edit it. I just want to tidy it up and have all the links in a single list for copy-n-paste/reference later.

PS1: I'll try to re-enable the sync in some of those but I already notice I don't have the proper rights in some of them. I'll let you know which ones.

**Comment by mwengren** [comment]
@ocefpaf yes, definitely copy my list on to yours.  

It looks like it ran via daily cron yesterday, at least in the case of ATN: https://github.com/ioos/ioos-atn-data/commits/gh-pages.  

I only went as far as re-enabling the workflow assuming cron would auto-start it, or so I thought.  Did you have to go back to those repos to make additional changes?  

Thanks!

**Comment by ocefpaf** [comment]
> I only went as far as re-enabling the workflow assuming cron would auto-start it, or so I thought. Did you have to go back to those repos to make additional changes?

You are correct but then you need to wait for the cronjob to kick in. I usually just hit the workflow dispatch to fix the sire navbar ASAP.
